packages/@aws-cdk/aws-apigatewayv2-integrations/lib/http/service-discovery.ts

```diff
@@ -9,7 +9,7 @@ export interface HttpServiceDiscoveryIntegrationProps {
   /**
```

Comment:
Do you think we should make this interface similar to the other integrations (where we expect a vpc to be passed and create the vpc-link on basis of that)? This will make the interfaces consistent. Thoughts?

Reply:
Good question. It made me realize my approach was all wrong. I had missed the fact that the props extended from `HttpPrivateIntegrationOptions`.

Take a look at the updated code. It should be better.